src/traces/box/cross_trace_calc.js

```diff
@@ -213,6 +212,7 @@ function setPositionOffset(traceType, gd, boxList, posAxis) {
             padded: padded,
             vpadminus: vpadminus,
             vpadplus: vpadplus,
+            vpadLinearized: true,
```

Comment:
So, could you explain why we _need_ to pad in linear space (`l`) for log axes as opposed to in calc space (`c`)? 

Padding in calc space on log axes seems to work ok in other scenarios like scatter-marker traces.

Reply:
I think paddings in scatter-marker traces are not based on values (`vpad`), but pixels (`ppad`). So the padding is not affected by axes types.
Another example that uses values as margins is box traces, but box traces should be based on `c` values rather than `l` values.